cosmicfishpie/configs/default_survey_specifications/Euclid-ISTF-Spectroscopic-Pessimistic.yaml

```diff
@@ -0,0 +1,5 @@
+specifications:
+  'spec_sigma_dz': 0.001
+  'area_survey': 15000
```

Comment:
can you change this entry to area_survey_spectro, so I can run directly on this branch when merged?